LondonBritishLibrary/orient/BLorient5268.xml

```diff
@@ -0,0 +1,400 @@
+                            </layoutDesc>    
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Careful handwriting.</desc>
+                                <date notBefore="1700" notAfter="1800" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="158v" to="159r"/>
+                                    <desc type="GuestText">Magical Prayer against charms and diseases</desc>
+                                    <q xml:lang="en">ስፍሐዝዝር፡ ፒሮስ፡ አልክምኑን፡ ኤፍታሔ፡ ስፍሐዝዝር፡ ፒሮስ፡ አልክምኑን፡ ኤፍታሔ፡ ፍታሕ፡ ኵሎ፡ ሥራየ፡ ወኵሉ፡ መቅጹተ፡ ዘተገብር፡ 
+                                        እምኀበ፡ አረሚ፡ አው፡ እምክርቲያን፡ <gap reason="ellipsis"/></q>
+                                </item>
+                            </list>
+                        </additions>           
+                    </physDesc>  
+                    </msPart>
+                    <msPart xml:id="p2">
+                        <msIdentifier></msIdentifier>
+                        <msContents><summary/>
+                            <msItem xml:id="p2_i1">
+                                <locus target="#160"/><locus target="#164"/>
+                                <title>Fragments of prayers</title>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent><measure unit="leaf">2</measure></extent>
+                                    <condition key="deficient">A part of <locus target="#160"/> is torn off and <locus target="#164"/> has been badly
+                                        damaged by a corrosive liquid.</condition>
+                                </supportDesc>
+                                <layoutDesc>
+                                    <layout columns="2"/>
+                                </layoutDesc>
+                            </objectDesc>  
+                        </physDesc>
+                    </msPart>
+                     <msPart xml:id="p3">
+                         <msIdentifier></msIdentifier>
+                         <msContents><summary/>
+                            <msItem xml:id="p3_i1">
+                                <locus from="161" to="162"/>
+                                <title ref="LIT4711Mastabqwe" type="incomplete">Deprecatory prayers</title>
```

Comment:
I am confused by "deprecatory". The prayers of LIT4711Mastabqwe, as I am "familiar" (not very) with them, are rather supplicative.

Reply:
I can delete the verbal title, because the LIT is sufficient.